### PR TITLE
Solved Issue #903 (Show/hide AM hours,PM hours,minutes)

### PR DESCRIPTION
**Pull request by sarthak-g**
Solved issue #903 

https://user-images.githubusercontent.com/42293606/102902267-c7a10800-4494-11eb-8a00-0708c9eec2cf.mp4

### CODE REVIEW

**Comment by llaske** [comment]
Very good job @sarthak-g. It's cool!

Few remarks:
- The new button should be to the right of the show/hide seconds. It's where all settings button are.
- The settings should be saved in the context so when the activity will be reopen, it will be reopen in the same state

Peek 22-12-2020 21-20

- One small screens, the Stop button is now invisible. It should always be visible to be able to quit the app.

Peek 22-12-2020 21-25

**Comment by sarthak-g** [comment]
@llaske made suggested changes. Please review.

https://user-images.githubusercontent.com/42293606/102982043-e86a6b80-452f-11eb-97be-3a0ddfe68a3a.mp4

**Review by llaske — CHANGES_REQUESTED** [review]
Issue on context is now solved.
Regarding responsiveness issue your solution work but it's not the standard solution for Sugarizer activities.

File: activities/Clock.activity/css/activity.css [hunk 1]
```diff
@@ -233,3 +233,12 @@ canvas {
     right: 15%;
   }
 }
+
+@media only screen and (max-width: 780px) {
+  .toolbar .toolbutton::before {
+    right: -8px;
+  }
+  .toolbar .toolbutton {
+    width: 36px;
```
[the remark this answers is not in the document]
Comment (sarthak-g) [reply activities/Clock.activity/css/activity.css]: In the latest commit, the hours and minutes display button is hidden for small screen size

File: activities/Clock.activity/js/activity.js [hunk 2]
```diff
@@ -21,6 +21,8 @@ define(["sugar-web/activity/activity","sugar-web/env","sugar-web/graphics/radiob
             } else {
                 activity.getDatastoreObject().loadAsText(function(error, metadata, data) {
                     if (error==null && data!=null) {
+                        show_am_pm = data.show_am_pm;
+                        if(show_am_pm) document.getElementById("show-am-pm").classList.add("active");
```
[the remark this answers is not in the document]
Comment (sarthak-g) [reply activities/Clock.activity/js/activity.js]: Okay

**Comment by llaske** [comment]
Very good. Thanks.

**Pull request merged**